src/fontra/client/core/glyph-lines.js

```diff
@@ -106,7 +106,7 @@ export function textFromGlyphLines(glyphLines) {
     let textLine = "";
     for (let i = 0; i < glyphLine.length; i++) {
       const glyphInfo = glyphLine[i];
-      if (glyphInfo.character) {
+      if (glyphInfo.character && glyphInfo.character !== "/") {
```

Comment:
I prefer "/" to be inserted as "//" which is the character escape for "/":

```suggestion
      if (glyphInfo.character) {
       textLine += "//";
      } else if (glyphInfo.character) {
```

Reply:
(The original suggestion was messed up, I think I fixed it now.)